app/workers/gamedev_worker.rb

```diff
@@ -0,0 +1,19 @@
+class GamedevWorker
+	include Sidekiq::Worker
+
+	def perform
+		response = RubyStackoverflow.questions({ order: 'asc', filter: 'withbody', pagesize: 100 })
+		response.data.each do | question|
+		  p =  ExternalPost.new(title: question.title,
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not -1) spaces for indentation.<br>Tab detected.